config/codestyle/eclipse-formatter.xml

```diff
@@ -0,0 +1,413 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="23">
+    <profile kind="CodeFormatterProfile" name="candlepin" version="23">
+        <setting id="org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode" value="enabled"/>
+        <setting id="org.eclipse.jdt.core.compiler.codegen.targetPlatform" value="17"/>
```

Comment:
Do we have to specify the Java version here? Are we going to need to update this as we update which version(s) of Java we support?

Reply:
I wasn't sure whether it's needed, but it looks like it's not.